docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -409,69 +528,99 @@ endif::[]
         # Do not attempt OVS major upgrades here
         edpm_ovs_packages:
         - openvswitch3.1
+  nodes:
 EOF
+  cat computes-$CELL >> nodeset-${CELL}.yaml
+done
 ----
 +
-<1> If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
-<2> If your deployment has a custom DNS Domain, modify the `spec:nodes:[NODE NAME]:hostName` to use fqdn for the node.
-<3> Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+<1> The networks composition must match the source cloud configuration to avoid dataplane connectivity downtime. The ctlplane network must come first.
+<2> Use node sets names, like `openstack-cell1`, `openstack-cell2`. Only create node sets for cells containing compute nodes.
+<3> If TLS Everywhere is enabled, change `spec.tlsEnabled` to `true`.
+<4> If not adopting the telemetry services, omit it from the services list.
+<5> The bridge name and other OVN and Neutron specific values must match the source cloud configuration to avoid dataplane connectivity downtime.
+<6> Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+
+[NOTE]
+The global service `ssh-known-hosts` may only be defined for a single node set.
```

Comment:
I think you can drop this since https://github.com/openstack-k8s-operators/openstack-operator/commit/19f6d2cb49abbcd17f41f8fb5a2c2b8e81368bc6